templates/project/sdkconfig.defaults

```diff
@@ -17,6 +17,8 @@ CONFIG_ESPTOOLPY_FLASHMODE_DIO=n
 
 CONFIG_ESP_TLS_SERVER=y
 CONFIG_MBEDTLS_SSL_PROTO_DTLS=y
+CONFIG_MBEDTLS_CERTIFICATE_BUNDLE=y
+CONFIG_MBEDTLS_CERTIFICATE_BUNDLE_DEFAULT_CMN=y
```

Comment:
I didn't see any `MBEDTLS_CERTIFICATE` values in the template. not sure if they were left out on purpose or not

Reply:
Definitely add them.

Would you mind filing a follow-up ticket to examine and resolve the differences between the sdkconfig for the viam-micro-server and what is in the template?

Reply:
[RSDK-10363](https://viam.atlassian.net/browse/RSDK-10363?atlOrigin=eyJpIjoiMTNlYzIzYWYxNDMyNDIwZjhjMDczM2M5OTBhMjJkM2IiLCJwIjoiaiJ9)